libraries/rush-sdk/src/index.ts

```diff
@@ -4,6 +4,8 @@
 import * as path from 'path';
 import { Import, IPackageJson, PackageJsonLookup } from '@rushstack/node-core-library';
 
+const RUSH_LIB_NAME: string = '@microsoft/rush-lib';
```

Comment:
```suggestion
const RUSH_LIB_NAME: '@microsoft/rush-lib' = '@microsoft/rush-lib';
```

Reply:
Not fixing -- this seems more verbose without serving any useful purpose